aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -272,21 +284,19 @@ def file_format_verbose(self) -> str:
     def export_version(self) -> str:
```

Comment:
should this move to the abstract class?

Reply:
The idea is that the abstract class is publicly available, such that (in principle) anyone can make their own archiver by implementing a writer and reader (to come) which would have their own versions.
The `EXPORT_VERSION` in aiida I feel is specific only to the current JSON archivers.
(still have to look into how this relates to migrations though)